pkg/tekton/metapipeline/metapipeline.go

```diff
@@ -164,9 +170,18 @@ func stepEffectivePipeline(params CRDCreationParameters) syntax.Step {
 func stepCreateTektonCRDs(params CRDCreationParameters) syntax.Step {
 	args := []string{"--clone-dir", filepath.Join(tektonBaseDir, params.SourceDir)}
 	args = append(args, "--build-number", params.BuildNumber)
+	args = append(args, "--trigger", params.Trigger)
```

Comment:
+1, I guess now everything is explicitly passed what is needed for step_create_task. 

I still "somehow" would like to get away from this args building stuff, but not quite yet sure how. For now this makes  most sense.